docs-new/docs/graphql/core/remote-schemas/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+description: Manage remote schemas with Hasura
+keywords:
+  - hasura
+  - docs
+  - remote schema
+slug: index
+sidebar_position: 1
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Remote Schemas
+
+## Introduction
+
+Hasura gives you CRUD + realtime GraphQL APIs with authorization & access control. However, in many cases, you will need to write APIs (queries, mutations) that contain custom logic. For example, implementing a payment API, or querying data that is not in your database.
+
+Hasura has the ability to merge remote GraphQL schemas and provide a unified GraphQL API. Think of it like automated schema stitching. All you need to do is build your own GraphQL service and then provide the HTTP endpoint to Hasura. Your GraphQL service can be written in any language or framework.
+
+This is what Hasura running with "Remote schemas" looks like:
+
+<Thumbnail
+  src="/img/graphql/core/remote-schemas/remote-schema-arch.png"
+  alt="Hasura scheduled trigger architecture"
+  width="900px"
+/>
+
+## Use cases
+
+- Custom business logic, like a payment API
+- Querying data that is not available in your database
+
+You can handle these use cases by writing resolvers in a custom GraphQL server and making Hasura merge this "remote schema" with the existing auto-generated schema. You can also add multiple remote schemas. Think of the merged schema as a union of top-level nodes from each of the sub-schemas.
+
+:::info Note
+
+If you are looking for adding authorization & access control for your app users to the GraphQL APIs that are auto-generated via Hasura, head to [auth](/graphql/core/remote-schemas/auth/index.mdx).
```

Comment:
Mismatching `auth` link and label.

Please refer live for correct label and link.

Link - `/graphql/core/auth/index.mdx`
Label - `Authentication & Authorization`
```suggestion
If you are looking for adding authorization & access control for your app users to the GraphQL APIs that are auto-generated via Hasura, head to [Authentication & Authorization](/graphql/core/auth/index.mdx).
```

Reply:
done